types/networks/src/lib.rs

```diff
@@ -242,19 +202,15 @@ impl ChainConfig {
     }
 
     pub fn network_version(&self, epoch: ChainEpoch) -> NetworkVersion {
-        let height = self
-            .height_infos
+        let height_infos = sort_by_epoch(&self.height_infos);
```

Comment:
Just a question, isn't it possible to sort by epoch at config creation time, so only once?

Reply:
As we are getting Config from file, it is not

Reply:
Ah I see, thanks.